README.md

```diff
@@ -4,7 +4,15 @@ A python package which brings better docstrings with an impeccable bedside manne
 
 The `attending` supervises other docs, and has the last word.
 
-## Improving the quality of resident docs.
+## Using
+
+coming soon: `pip install attending`
```

Comment:
I'm sure ill forget to update this later so maybe we should leave this out for now?